### PR TITLE
Explicitly handle Archivematica Archivesspace cfg

### LINKED ISSUE #967: Problem: Dashboard shouldn't need to try/except to test existence of ArchivesSpace
Monitoring the logs of the dashboard shows that this error appears:
```
archivematica-dashboard_1        | ERROR     2018-03-09 07:52:43  django.request:base:handle_uncaught_exception:256:  Internal Server Error: /access/archivesspace/levels/
archivematica-dashboard_1        | Traceback (most recent call last):
archivematica-dashboard_1        |   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
archivematica-dashboard_1        |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
archivematica-dashboard_1        |   File "/src/dashboard/src/components/access/views.py", line 48, in wrapper
archivematica-dashboard_1        |     client = _get_client_by_type(system)
archivematica-dashboard_1        |   File "/src/dashboard/src/components/access/views.py", line 35, in _get_client_by_type
archivematica-dashboard_1        |     return get_as_system_client()
archivematica-dashboard_1        |   File "/src/dashboard/src/components/ingest/views_as.py", line 27, in get_as_system_client
archivematica-dashboard_1        |     repository=config['repository']
archivematica-dashboard_1        |   File "/usr/local/lib/python2.7/site-packages/agentarchives/archivesspace/client.py", line 61, in __init__
archivematica-dashboard_1        |     self._login()
archivematica-dashboard_1        |   File "/usr/local/lib/python2.7/site-packages/agentarchives/archivesspace/client.py", line 65, in _login
archivematica-dashboard_1        |     response = requests.post(self.host + '/users/' + self.user + '/login', data={'password': self.passwd, 'expiring': False}, timeout=self.timeout)
archivematica-dashboard_1        |   File "/usr/local/lib/python2.7/site-packages/requests/api.py", line 112, in post
archivematica-dashboard_1        |     return request('post', url, data=data, json=json, **kwargs)
archivematica-dashboard_1        |   File "/usr/local/lib/python2.7/site-packages/requests/api.py", line 58, in request
archivematica-dashboard_1        |     return session.request(method=method, url=url, **kwargs)
archivematica-dashboard_1        |   File "/usr/local/lib/python2.7/site-packages/requests/sessions.py", line 494, in request
archivematica-dashboard_1        |     prep = self.prepare_request(req)
archivematica-dashboard_1        |   File "/usr/local/lib/python2.7/site-packages/requests/sessions.py", line 437, in prepare_request
archivematica-dashboard_1        |     hooks=merge_hooks(request.hooks, self.hooks),
archivematica-dashboard_1        |   File "/usr/local/lib/python2.7/site-packages/requests/models.py", line 305, in prepare
archivematica-dashboard_1        |     self.prepare_url(url, params)
archivematica-dashboard_1        |   File "/usr/local/lib/python2.7/site-packages/requests/models.py", line 382, in prepare_url
archivematica-dashboard_1        |     raise InvalidURL("Invalid URL %r: No host supplied" % url)
archivematica-dashboard_1        | InvalidURL: Invalid URL u'http://:8089/users//login': No host supplied
```

`Invalid URL u'http://:8089/users//login`

It should be possible to detect sooner than this that the URL we supply is invalid, and it should be possible to detect outside of a try/except statement that this is the case.

### PR DESCRIPTION
**Pull request by ross-spencer**
Because we are able to identify in our own configuration that the host
string to conect to Archivesspace is still blank we know that it hasn't
been configured. In those instances, we can raise a custom exception
stating as much, instead of attempting to create an Archivesspace client
and handling an InvalidURL exception.

Resolves #967

### CODE REVIEW

File: src/dashboard/src/components/ingest/views_as.py
Comment (sevein) [inline src/dashboard/src/components/ingest/views_as.py]: You could make it a subclass of `ArchivesSpaceError` or just raise `ArchivesSpaceError` considering that the outcome is the same regardless.

If you decide to stick with `ArchivesSpaceConfigurationException` I think that you should probably call it `ArchivesSpaceConfigurationError` instead, that seems to be the naming convention.

**Review by sevein — APPROVED** [review]
This is nice, @ross-spencer. Small suggestion but once you fix it feel free to merge right away.

File: src/dashboard/src/components/ingest/views_as.py
Comment (ross-spencer) [reply src/dashboard/src/components/ingest/views_as.py]: I went for ArchivesSpaceError in the end! Good call.

**Comment by sevein** [comment]
a96fcbb2c088d9bf05cc3c60ed21823093e97571 in `stable/1.7.x`

**Pull request merged**
